pkgs/development/tools/cue/default.nix

```diff
@@ -2,12 +2,12 @@
 
 buildGoModule rec {
   pname = "cue";
-  version = "0.3.0";
+  version = "0.3.2";
 
   src = fetchgit {
     url = "https://cue.googlesource.com/cue";
     rev = "v${version}";
-    sha256 = "1h3809xgmn7dr57i3cnifr7r555i3zh3kfsv0gxa9nd7068w19xm";
+    sha256 = "0rfgpq4dyd3zm07vcjzn5vv0dhvvryrarxc50sd2pxagbq5cqc8l";
   };
 
   vendorSha256 = "10kvss23a8a6q26a7h1bqc3i0nskm2halsvc9wdv9zf9qsz7zjkp";
```

Comment:
Please add ``-s -m`` to buildFlagsArray.

Reply:
What are theses options for ?

Reply:
They strip debugging symbols.

Reply:
@SuperSandro2000  Seems it's `-s -w` for go, updated